koa-views/index.d.ts

```diff
@@ -0,0 +1,60 @@
+// Type definitions for koa-views v2.x
```

Comment:
Please add tslint.json to your project, this should catch this invalid version string. see https://github.com/DefinitelyTyped/DefinitelyTyped#lint for details